### PR TITLE
front: add lazy loader message on GET in stdcm

### LINKED ISSUE #8610: Missing lazy loader info in STDCM GET
### What happened?

This info is missing in STDCM GET
image

### What did you expect to happen?

_No response_

### How can we reproduce it (as minimally and precisely as possible)?

.

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

1482e8c

### PR DESCRIPTION
**Pull request by clarani**
closes #8610

image

### CODE REVIEW

File: front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx [hunk 1]
```diff
@@ -29,6 +30,8 @@ type ManchetteWithSpaceTimeChartProps = {
   waypointsPanelData?: WaypointsPanelData;
   conflicts?: Conflict[];
   workSchedules?: PostWorkSchedulesProjectPathApiResponse;
+  totalTrains: number;
+  allTrainsProjected: boolean;
```
Comment (SharglutDev) [inline front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx]: As they both work together and are only here for the projection loader message, should we put them in an object to have less props to pass across all components ?

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested, good job ! Left just one suggestion, you can resolve yourself after treating it or not :)

File: front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx (hunk 1 above)
Comment (clarani) [reply front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx]: Yes we can, but how would you name this object ?

Comment (SharglutDev) [reply front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx]: `ProjectionLoaderData` ? Like we do for other part of the simulation

**Pull request merged**